pinery-lims-miso/src/main/java/ca/on/oicr/pinery/lims/miso/MisoClient.java

```diff
@@ -146,70 +147,173 @@
-      + "        INNER JOIN Indices ON Indices.indexId = Library_Index.index_indexId\n"
-      + "        WHERE position = 1\n"
-      + "        ) bc1 ON bc1.library_libraryId = l.libraryId\n" + "LEFT JOIN (\n" + "        SELECT library_libraryId\n"
-      + "                ,sequence\n" + "        FROM Library_Index\n"
-      + "        INNER JOIN Indices ON Indices.indexId = Library_Index.index_indexId\n"
-      + "                WHERE position = 2\n" + "        ) bc2 ON bc2.library_libraryId = l.libraryId\n"
-      + "LEFT JOIN BoxPosition pos ON pos.boxPositionId = l.boxPositionId\n" + "LEFT JOIN Box box ON box.boxId = pos.boxId";
+  private static final String queryAllSamples = "SELECT DISTINCT s.alias NAME\n" + 
+      "        ,s.description description\n" + 
+      "        ,s.NAME id\n" + 
+      "        ,parent.NAME parentId\n" + 
+      "        ,sc.alias sampleType\n" + 
+      "        ,NULL sampleType_platform\n" + 
+      "        ,NULL sampleType_description\n" + 
+      "        ,tt.alias tissueType\n" + 
+      "        ,p.alias project\n" + 
+      "        ,sai.archived archived\n" + 
+      "        ,scl.creationDate created\n" + 
+      "        ,sclcu.userId createdById\n" + 
+      "        ,scl.lastUpdated modified\n" + 
+      "        ,scluu.userId modifiedById\n" + 
+      "        ,s.identificationBarcode tubeBarcode\n" + 
+      "        ,s.volume volume\n" + 
+      "        ,ss.concentration concentration\n" + 
+      "        ,s.locationBarcode storageLocation\n" + 
+      "        ,kd.NAME kitName\n" + 
+      "        ,kd.description kitDescription\n" + 
+      "        ,s.receivedDate receive_date\n" + 
+      "        ,i.externalName external_name\n" + 
+      "        ,tor.alias tissue_origin\n" + 
+      "        ,tm.alias tissue_preparation\n" + 
+      "        ,st.region tissue_region\n" + 
+      "        ,st.tubeNumber tube_id\n" + 
+      "        ,ss.strStatus str_result\n" + 
+      "        ,sai.groupId group_id\n" + 
+      "        ,sai.groupDescription group_id_description\n" + 
+      "        ,sp.alias purpose\n" + 
+      "        ,qubit.results qubit_concentration\n" + 
+      "        ,nanodrop.results nanodrop_concentration\n" + 
+      "        ,NULL barcode\n" + 
+      "        ,NULL barcode_two\n" + 
+      "        ,qpcr.results qpcr_percentage_human\n" + 
+      "        ,s.qcPassed qcPassed\n" + 
+      "        ,qpd.description qcPassedDetail\n" + 
+      "        ,box.locationBarcode boxLocation\n" + 
+      "        ,box.alias boxAlias\n" + 
+      "        ,pos.row boxRow\n" + 
+      "        ,pos.COLUMN boxColumn\n" + 
+      "        ,NULL paired\n" + 
+      "        ,NULL read_length\n" + 
+      "        ,NULL targeted_resequencing\n" + 
+      "FROM Sample s\n" + 
+      "LEFT JOIN DetailedSample sai ON sai.sampleId = s.sampleId\n" + 
+      "LEFT JOIN QcPassedDetail qpd ON qpd.qcPassedDetailId = sai.qcPassedDetailId\n" + 
```

Comment:
This table will be renamed to `DetailedQcStatus` in next week's MISO release. Should the table name be changed now or after the release?